extensions/blocks/membership-button/edit.jsx

```diff
@@ -146,6 +146,7 @@ class MembershipsButtonEdit extends Component {
 						},
 					] ),
 				} );
+				this.setMembershipAmount( result.id );
```

Comment:
Would be good to add a comment here for the future as an intended UX.

Reply:
Added a comment now :)